api/src/main/java/org/pmiops/workbench/db/model/Workspace.java

```diff
@@ -85,6 +85,7 @@ public boolean equals(Object obj) {
   private Short activeStatus;
   private Short billingMigrationStatus =
       StorageEnums.billingMigrationStatusToStorage(BillingMigrationStatus.OLD);
+  private boolean published;
```

Comment:
We wanted to filter these out of getWorkspaces - if that's not meant to be in this PR, is that work tracked separately?

Reply:
unclear; i will check with karthik

Reply:
published workspaces should not show up unless user is explicitly a reader/writer/owner on the ws.  changed getWorkspaces() to reflect that.